npm/src/compiler/compiler.ts

```diff
@@ -43,14 +53,32 @@ export class Compiler implements ICompiler {
     globalThis.qscGitHash = this.wasm.git_hash();
   }
 
-  async checkCode(code: string): Promise<VSDiagnostic[]> {
+  async checkCode(
```

Comment:
I think I'd leave these out of the `checkCode` signature and just pass the defaults to the constructor below... This function is used to quickly check a piece of Q# code, and doesn't really make sense to use in a project scenario. (I may be proven wrong in the future when we use this with projects, but maybe we can just add the parameters then?)